src/sentry/snuba/metrics/query_builder.py

```diff
@@ -426,6 +426,29 @@ def parse_query(query_string: str, projects: Sequence[Project]) -> Sequence[Cond
     return where
 
 
+def parse_query_params(query_params, projects: Sequence[Project]) -> Sequence[Condition]:
```

Comment:
Why not add this logic into the `parse_query` function?

Reply:
I considered that at first, but `parse_query` is only called when there's a query provided, otherwise it's bypassed. I could update it and have it called every time. I wasn't sure which we'd prefer

Reply:
I've consolidated the logic and renamed `parse_query` to `parse_conditions`, let me know if you'd prefer this!